schemas/stsci.edu/asdf/time/time-1.1.0.yaml

```diff
@@ -0,0 +1,262 @@
+              optional if the `jyear` format is explicitly
+              specified.
+
+            - `decimalyear`: Time as a decimal year, with integer
+              values corresponding to midnight of the first day of
+              each year. For example 2000.5 corresponds to the ISO
+              time `2000-07-02 00:00:00`.
+
+            - `jd`: Julian Date time format. This represents the
+              number of days since the beginning of the Julian
+              Period. For example, 2451544.5 in `jd` is midnight on
+              January 1, 2000.
+
+            - `mjd`: Modified Julian Date time format. This
+              represents the number of days since midnight on
+              November 17, 1858. For example, 51544.0 in MJD is
+              midnight on January 1, 2000.
+
+            - `gps`: GPS time: seconds from 1980-01-06 00:00:00 UTC
+              For example, 630720013.0 is midnight on January 1,
+              2000.
+
+            - `unix`: Unix time: seconds from 1970-01-01 00:00:00
+              UTC. For example, 946684800.0 in Unix time is midnight
+              on January 1, 2000.  [TODO: Astropy's definition of
+              UNIX time doesn't match POSIX's here.  What should we
+              do for the purposes of ASDF?]
+
+          enum:
+            - iso
+            - yday
+            - byear
+            - jyear
+            - decimalyear
+            - jd
+            - mjd
+            - gps
+            - unix
+            - cxcsec
+
+        scale:
+          description: |
+            The time scale (or time standard) is a specification for
+            measuring time: either the rate at which time passes; or
+            points in time; or both. See also [3] and [4].
+
+            These scales are defined in detail in [SOFA Time Scale and
+            Calendar Tools](http://www.iausofa.org/sofa_ts_c.pdf).
+
+            The supported time scales are:
+
+            - `utc`: Coordinated Universal Time (UTC).  This is the
+              default time scale, except for `gps`, `unix`.
+
+            - `tai`: International Atomic Time (TAI).
+
+            - `tcb`: Barycentric Coordinate Time (TCB).
+
+            - `tcg`: Geocentric Coordinate Time (TCG).
+
+            - `tdb`: Barycentric Dynamical Time (TDB).
+
+            - `tt`: Terrestrial Time (TT).
+
+            - `ut1`: Universal Time (UT1).
+
+          enum:
+            - utc
+            - tai
+            - tcb
+            - tcg
+            - tdb
+            - tt
+            - ut1
+
+        location:
+          description: |
+            Specifies the observer location for scales that are
+            sensitive to observer location, currently only `tdb`.  May
+            be specified either with geocentric coordinates (X, Y, Z)
+            with an optional unit or geodetic coordinates:
+              - `long`: longitude in degrees
+              - `lat`: in degrees
+              - `h`: optional height
+
+          anyOf:
```

Comment:
Is this `anyOf` necessary here, given that there's only one option. Or am I misreading it?

Reply:
Good catch, I think this was just an oversight.